### ISSUE TITLE
Adaptive card collapses in message area when not in focus

### ISSUE THREAD

**Issue by ChetanSharma-msft** [MicrosoftDocs/msteams-docs#11462]
### Steps to reproduce

Copied bug from: https://github.com/OfficeDev/teams-toolkit/issues/12205

**Details from original post:
Describe the bug**

We have just published a new app to Microsoft Teams store. The app utilizes message extension feature and a bot from teams toolkit example. First users are complaining about something that most likely is a bug in the Teams app. You can reproduce the issue with our app: https://tinyurl.com/teams-issue-example-app . The link will expire automatically on 31th August 2024. Please do not share any screenshots of our app due to privacy. Thanks!

Our app gathers input from the user and returns an adaptive card for the message area. The user can then post the adaptive card to the chat. Under certain circumstances (see below), the adaptive cards collapses inside the message area and the user is forced to key in the data again. The users are already complaining to us, but it seems that this is a Teams bug that need to be fixed by Microsoft.

To Reproduce

Positive case (when it works):

Select the message area so that it has the focus (purple line at bottom).
image

Add an app via '+' in message area that utilizes message extension and returns an adaptive card for the compose area (like our app):
image

Adaptive card is shown in compose area. The user can submit the message:
image

Everything works fine in this case.

Negative case:

Click the '+' button in the message area. The text box does not have the focus (no purple line at bottom):
image

Add the app that return an adaptive card (like our app).
image

The adaptive card can be seen for < 1 sec and then it collapses in the compose area. The user has to start again. (Note: Sometimes you have to repeat this scenario 2-3 times to see the effect.)
Our app produces a very powerful adaptive card (universal adaptive card) with refresh capabilities. Maybe the preview in the compose area can not handle this when there is no focus? We have no tested this with other apps as we don't know which apps also produce adaptive cards via message extension.

### Expected behavior

The adaptive card must not collapse / disappear in the compose area when the control does not have the focus.

### Actual behavior

The adaptive card collapses / disappears in the compose area when the control does not have the focus. This leads to a bad user experience for our users and forces them to gather their data again. In worst case, it can lead to users stop using our app.

VS Code Extension Information (please complete the following information):

Teams Web (Edge)
Teams Desktop (Windows 10)

**Comment by Meghana-MSFT** [comment]
Thank you for reporting this, we will check this and get back to you.

**Comment by jker-org** [comment]
@Meghana-MSFT : We found a similar issue on Github. Maybe you connect with your collegues from team-js: https://github.com/OfficeDev/microsoft-teams-library-js/issues/2261

This issue is really affecting our business.

**Comment by Meghana-MSFT** [comment]
@jker-org - We checked this issue at our end, and we see that the issue occurs to us alternatively. I mean after the card is added to compose box, I can post the card to chat. When I try to create a poll again and preview it in the compose box, card disappears. If I try again, it works 3rd time and then card disappears 4th time and so on.

**Comment by Meghana-MSFT** [comment]
Could you please let us know if you also see any such pattern in the issue?

**Comment by jker-org** [comment]
@Meghana-MSFT : That is great. At least you can reproduce :-) Let me know if there is anything where we can help.

**Comment by jker-org** [comment]
> Could you please let us know if you also see any such pattern in the issue?

Yes, we confirm. It is really random. Sometimes first card fails. Sometimes first card succeeds. Sometimes we have to try 2 or 3 times and then it fails. It is hard to predict, but it happens very often.

**Comment by Meghana-MSFT** [comment]
@jker-org - We raised a bug for this, we will keep you posted on the updates. Thank you.

**Comment by jker-org** [comment]
@Meghana-MSFT : Thank you very much. From our perspective, the severity is high. Let us know if you need any help on this.

**Comment by jker-org** [comment]
Hi @Meghana-MSFT, is there any timeline for the fix? This issue is costing us a lot of users every day.

**Comment by Meghana-MSFT** [comment]
Hi @jker-org - Unfortunately, we do not have any ETA to share. We are following up with engineering team for updates. We will keep you posted.

**Comment by jker-org** [comment]
@Meghana-MSFT : Thank you very much. Is there any known workaround until the issue is fixed?

**Comment by Jocelynncanzoneri** [comment]
Yal why can't I work or make money with my phone. Drive app no. Amazon app
no.  I have to make money.  Can you show me a way to make money.  Emergency
cat needs to go to the vet.  My car headlights out.
Please thank you.

**Comment by Meghana-MSFT** [comment]
@jker-org - This issue has been fixed, we have verified the fix in Teams web (version is 1415/24110115716).

**Comment by jker-org** [comment]
@Meghana-MSFT: Thank you! We will check.
